orchagent/portsorch.cpp

```diff
@@ -1108,6 +1295,18 @@ bool PortsOrch::initializePort(Port &p)
     vector.push_back(tuple);
     m_portTable->set(p.m_alias, vector);
 
+    /*
+     * Port has been removed from 1q bridge at PortsOrch constructor,
+     * also start stipping off VLAN tag.
+     */
+    bool rv = setHostIntfsStripTag(p, SAI_HOSTIF_VLAN_TAG_STRIP);
```

Comment:
move this piece of code inside the `addHostIntfs` function？

Reply:
ok

Reply:
After checking it again, bool PortsOrch::setHostIntfsStripTag(Port &port, sai_hostif_vlan_tag_t strip) is taking Port class as first parameter,  while bool PortsOrch::addHostIntfs(sai_object_id_t id, string alias, sai_object_id_t &host_intfs_id) is using port id of type sai_object_id_t .  

Probably it is better to keep setHostIntfsStripTag() out side of addHostIntfs(), and leave setHostIntfsStripTag() at initializePort().

Reply:
what shall we do if this function fails here? what about moving this function above under the `addHostIntfs` function without checking the return value to match the rest of the initialization functions. it seems that if this function fails, there's nothing we could do here. since there's already error message inside this function, we could anyway catch it.

Reply:
ok,  will move the function to addHostIntfs() without checking return value.